doc.cfm

```diff
@@ -2,12 +2,13 @@
 <cfset url.name = ReReplace(url.name, "[^a-zA-Z0-9_-]", "", "ALL")>
 <cfif url.name IS "index">
 	<cfset data = {name="CFDocs", description="Ultra Fast CFML Documentation", type="index"}>
-<cfelseif FileExists(ExpandPath("./guides/en/#url.name#.md"))>
+<cfelseif FileExists(ExpandPath("./guides/en/#url.name#.md")) && url.name is "how-to-contribute">
```

Comment:
This change would cause all guides to fail /except how-to-contribute right?

Reply:
Silly me. I changed it.